src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Perhaps it is better to list out explicitly what classes you have imported from package?

Reply:
You're right, I should have listed out all of the commands explicitly for easier readability and future maintenance.